src/pages/community/index.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql } from 'gatsby';
+import { Box, Heading, Text, Image } from 'grommet';
+
+import { CommunityCard, Layout, SEO } from '../../components';
+import { useSiteMetadata } from '../../hooks/use-site-metadata';
+
+Heading.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+function Community({ data }) {
+  const communities = data.allMarkdownRemark.edges;
+  const siteMetadata = useSiteMetadata();
+  const siteTitle = siteMetadata.title;
+  return (
+    <Layout title={siteTitle}>
+      <SEO title="Community" />
+      <Box
+        flex
+        overflow="auto"
+        gap="medium"
+        pad="large"
+        direction="column"
+        wrap
+      >
+        <Box direction="row-responsive" align="start" gap="medium">
```

Comment:
We should make this top heading part a component since it's used in a bunch of pages

Reply:
I created a separate component for the heading part called the "PageDescription" component.